### DISCUSSION TITLE
有需要限制数据库对象，表，字段，索引长度不超过30字节

### DISCUSSION THREAD

**Discussion by planet-zjx** [hhyo/Archery#1686] [Q&A]
有需要限制数据库对象，表，字段，索引长度不超过30字节，翻了相关文档，只有一句”表名、列名、索引名的长度不大于64个字节“，自己实测超过64字节SQL检查会报错，没找到对应配置项，是否可以提供该配置项，以及该配置项是否已废除。

麻烦大佬，帮忙看下这个问题。

**Comment by hhyo** [comment]
审核规则参考 https://hanchuanchuan.github.io/goInception/zh/rules.html#ddl%E6%93%8D%E4%BD%9C

并不支持配置阈值

**Reply by planet-zjx** [reply]
多谢
